wallet/src/main/scala/org/bitcoins/wallet/models/OutgoingTransactionDAO.scala

```diff
@@ -0,0 +1,61 @@
+package org.bitcoins.wallet.models
+
+import org.bitcoins.core.crypto.{DoubleSha256Digest, DoubleSha256DigestBE}
+import org.bitcoins.db.{CRUD, SlickUtil}
+import org.bitcoins.wallet.config._
+import slick.jdbc.SQLiteProfile.api._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+case class OutgoingTransactionDAO()(
```

Comment:
rather than duplicate a lot of code across `IncomingTransactionDAO()`/`OutgoingTransactionDAO()`/`TransactionDAO()` why not make an abstract class for the similar functionality and then extend it in these things?